### PR TITLE
Fix/marketo/update to new look and feel

### PR DESCRIPTION
**Pull request by ricardo-valente**
- applied theme visualization to form elements

### CODE REVIEW

File: packages/molecules/form-marketo/package.prd.json [hunk 1]
```diff
@@ -29,6 +29,9 @@
   },
   "dependencies": {
     "@creat/common": "^1.5.0",
-    "@creat/styles": "^1.5.0"
+    "@creat/styles": "^1.5.0",
+    "@creat/buttons": "^1.5.0",
```
Comment (rebecatalkdesk) [inline packages/molecules/form-marketo/package.prd.json:33]: Why did you  add the buttons, dropdown and inputs as dependency? They're not being used anywhere...

Comment (ricardo-valente) [reply packages/molecules/form-marketo/package.prd.json:33]: we have a dependency on styles

Comment (ricardo-valente) [reply packages/molecules/form-marketo/package.prd.json:33]: if do not include the input styles here, we'll have input styles present in the DOM?

Comment (ricardo-valente) [reply packages/molecules/form-marketo/package.prd.json:33]: since all styles are already present in the DOM, I'll remove those from dependencies

**Review by rebecatalkdesk — APPROVED** [review]

**Pull request merged**